### DISCUSSION TITLE
Media Interview

### DISCUSSION THREAD

**Discussion by oldgristmill** [mcguirepr89/BirdNET-Pi#257] [Show and tell]
Hey @mcguirepr89 (and others), I just got a call from a local reporter about our own BirdNET-Pi installation after I published a short blog post about it for the heritage site I manage: https://www.oldgristmill.ca/blog/project-birdbox/.

She asked all sorts of great questions and I hope she includes in her piece the generous praise I've expressed about everyone contributing to this project. Maybe it'll get a few new people interested in this exciting project.

I'll be sure to post a link to the article when it gets published..

**Comment by mcguirepr89** [comment]
@oldgristmill Woah!!! So cool!!!!!!!!!!! Thank you so much! I'm really hoping that it does just as you say and that folks will embrace the project the way you (we all) have!

I look forward to reading the article and learning more about Old Grist Mill :)

**Comment by oldgristmill** [comment]
The article is out and it... could have been so much more. Oh well... still cool to see it mentioned.

https://infotel.ca/newsitem/okanagan-heritage-farm-using-high-tech-audio-device-to-track-and-record-birds/it90576?

**Comment by avit** [comment]
My wife read that article and asked me "isn't that the same thing you are doing"... good job!

I'm at the northern end of the valley from you in Vernon, and hope to have our own installation running soon. We have a good diversity of birds around our property so it will be fun to compare.

**Comment by avit** [comment]
This is my station by the way: https://app.birdweather.com/stations/238

I haven't opened it directly to the internet yet.
